ironfish/src/primitives/block.ts

```diff
@@ -98,15 +96,9 @@ export type SerializedBlock = {
 
 export type SerializedCounts = { notes: number; nullifiers: number }
 
-export class BlockSerde implements Serde<Block, SerializedBlock> {
-  blockHeaderSerde: BlockHeaderSerde
-
-  constructor(readonly strategy: Strategy) {
-    this.blockHeaderSerde = new BlockHeaderSerde(strategy)
-  }
-
-  equals(block1: Block, block2: Block): boolean {
-    if (!this.blockHeaderSerde.equals(block1.header, block2.header)) {
+export class BlockSerde {
```

Comment:
this part is a bit odd because we're mixing serializer patterns now I guess. It looks like a few others were already made static, and serdeInstances are the other ones that are still instances.

Reply:
That's a good question. I think there's one more step to be done here, probably removing these "Serde" classes that no longer implement the `Serde` interface, and moving the methods to their respective classes. The remaining ones actually use the `Serde` generic bits for the merkle tree stuff which is something that should be a separate PR.